qa/testdata/listing_crypto.json

```diff
@@ -15,6 +15,12 @@
         "description": "",
         "processingTime": "1 to 2 hours",
         "price": "0",
+        "priceCurrency": {
+            "code": "TETH",
+            "divisibility": 8,
+            "name": "A",
+            "currencyType": "A"
```

Comment:
`Name` and `currencyType` aren't persisted in the actual json payloads. We can leave them out of the fixtures.